reviewing/plugin-commands-licenses/src/outputRenderer.ts

```diff
@@ -112,14 +112,53 @@ function renderLicensesTable (
   for (let i = 0; i < columnNames.length; i++)
     columnNames[i] = chalk.blueBright(columnNames[i])
 
-  return table(
-    [
-      columnNames,
-      ...deduplicateLicensesPackages(sortLicensesPackages(licensePackages))
-        .map((licensePkg) => columnFns.map((fn) => fn(licensePkg))),
-    ],
-    TABLE_OPTIONS
-  )
+  const data = [
+    columnNames,
+    ...deduplicateLicensesPackages(sortLicensesPackages(licensePackages))
+      .map((licensePkg) => columnFns.map((fn) => fn(licensePkg))),
+  ]
+  let detailsColumnMaxWidth = 40
+  let packageColumnMaxWidth = 0
+  let licenseColumnMaxWidth = 0
+  if (opts.long) {
+    // Use the package link to determine the width of the details column
+    detailsColumnMaxWidth = licensePackages.reduce((max, pkg) => Math.max(max, pkg.homepage?.length ?? 0), 0)
+    for (let i = 1; i < data.length; i++) {
+      const row = data[i]
+      const texts = row[2].split('\n')
+      const repeatCount = Math.max(0, texts.length - 1)
```

Comment:
maybe rename it to linesNumber?
```suggestion
      const detailsLineCount = row[2].split('\n').length
      const repeatCount = Math.max(0, detailsLineCount - 1)
```

Reply:
Done.